app/assets/javascripts/bike_index.js.coffee

```diff
@@ -106,6 +106,7 @@ $(document).ready ->
   else if document.getElementById('admin-content')
     new BikeIndex.Views.DataTables
     if document.getElementById('bike_edit_root_url')
+      # TODO: Is this dead code?
```

Comment:
Yes! Good catch.

We also have a bunch of legacy backbone that is lying around. I think the admin bikes edit revised coffee is good to delete, I'm waiting to delete the backbone stuff until we've fully finished updating the admin.